infrastructure/Jenkinsfile-CI.groovy

```diff
@@ -28,6 +28,14 @@ pipeline {
                 '''
             }
         }
+        stage('Build Keycloak extensions') {
+            steps {
+                sh'''
+                    docker build --pull --build-arg JENKINS_USER_ID=$(id -u jenkins) --build-arg JENKINS_GROUP_ID=$(id -g jenkins) -t ${DOCKER_PROJECT_NAME}-maven:${BUILD_NUMBER} infrastructure/docker/maven
+                    docker run --rm -v $PWD:/code -w /code/registration-event-listener ${DOCKER_PROJECT_NAME}-maven:${BUILD_NUMBER} mvn -B -U clean install
```

Comment:
I think you still have to execute it with the Jenkins user: https://github.com/noi-techpark/davinci-innovation-scoreboard-api/blob/master/infrastructure/Jenkinsfile-Test.groovy#L79